### DISCUSSION TITLE
这地平线真是一件美事啊~

### DISCUSSION THREAD

**Discussion by Boring-Coils** [share121/inter-knot#2985] [其他]
![微信图片_20240819094737](https://github.com/user-attachments/assets/95e7af0a-145e-41dc-bf94-8e8d86fe6f63)
![微信图片_20240819094729](https://github.com/user-attachments/assets/9abb0808-ed38-4acf-8cb3-743804fb7665)
![微信图片_20240819094724](https://github.com/user-attachments/assets/0e7c40fe-5435-4032-bf42-3c269497a4bc)
![微信图片_20240819094720](https://github.com/user-attachments/assets/05f608bd-8e53-44d8-a177-acb483562a77)
![微信图片_20240819094717](https://github.com/user-attachments/assets/0981d43b-3b66-4fad-a8ac-1f97e571ba3e)
![微信图片_20240819094714](https://github.com/user-attachments/assets/32e8c9ff-db0e-408a-aad4-6569af6adf38)
![微信图片_20240819094711](https://github.com/user-attachments/assets/473198ac-863b-4791-9f2c-efd2f816626a)
![微信图片_20240819094708](https://github.com/user-attachments/assets/939325b0-8134-46c1-8af6-d08b689898f0)
![微信图片_20240819094702](https://github.com/user-attachments/assets/6e7fcf28-5c45-40bf-9d20-398386229bff)

**Comment by Boring-Coils** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/fe198bb1523528c2b852d28626379a2a.png" alt="心情简单" width="75" >

**Comment by Loxic123** [comment]
<img width="1280" alt="屏幕截图 2024-07-20 004035" src="https://github.com/user-attachments/assets/679b4d08-99c5-4198-b5dc-67accfafd180">
<img width="1280" alt="屏幕截图 2024-07-20 004057" src="https://github.com/user-attachments/assets/c237d0d0-794d-4928-aa17-cc5cbe1b32b2">
![20240719230925_1](https://github.com/user-attachments/assets/d5311fff-704b-4ec6-ad56-804e24e94473)
![20240719230904_1](https://github.com/user-attachments/assets/fe3b13a9-8d2b-492e-af23-fe3b57075235)
二次元是这样的，就算是开车也会弄个二次元皮<img src="https://bbs-static.miyoushe.com/static/2024/01/31/a065264ccf369d43f39162320393d9bf_5251148779871794599.png" alt="花火 我来啦" width="75" >

**Comment by Loxic123** [comment]
![20240719230904_1](https://github.com/user-attachments/assets/2a992bf7-11a9-4657-9650-f71ee1288527)
![20240719230925_1](https://github.com/user-attachments/assets/8fbb1d2b-f118-454e-aacc-33b169419ab8)
<img src="https://bbs-static.miyoushe.com/static/2024/06/19/52971140330611d82cad599f20788ab6_3848320906174398412.png" alt="帕姆 开拓" width="75" >

**Comment by Fluorine-z** [comment]
我也想入地平线，但一直在想...

**Comment by Junjun280** [comment]
是挺不戳的

**Comment by miheihuyou** [comment]
牛哇牛哇
